res/swars-setup.nsi

```diff
@@ -776,228 +815,7 @@ Delete '$INSTDIR\data\equip\WEP-29Z.FLI'
-Delete '$INSTDIR\levels\C040L013.DAT'
-Delete '$INSTDIR\levels\C040L015.D1'
-Delete '$INSTDIR\levels\C040L015.D2'
-Delete '$INSTDIR\levels\C040L015.DAT'
-Delete '$INSTDIR\levels\C041L001.D1'
-Delete '$INSTDIR\levels\C041L001.D2'
-Delete '$INSTDIR\levels\C041L001.D3'
-Delete '$INSTDIR\levels\C041L001.DAT'
-Delete '$INSTDIR\levels\C041L002.D1'
-Delete '$INSTDIR\levels\C041L002.D2'
-Delete '$INSTDIR\levels\C041L002.D3'
-Delete '$INSTDIR\levels\C041L002.DAT'
-Delete '$INSTDIR\levels\C041L003.DAT'
-Delete '$INSTDIR\levels\C041L005.DAT'
-Delete '$INSTDIR\levels\C044L001.D1'
-Delete '$INSTDIR\levels\C044L001.D2'
-Delete '$INSTDIR\levels\C044L001.D3'
-Delete '$INSTDIR\levels\C044L001.DAT'
-Delete '$INSTDIR\levels\C044L002.D1'
-Delete '$INSTDIR\levels\C044L002.D2'
-Delete '$INSTDIR\levels\C044L002.DAT'
-Delete '$INSTDIR\levels\C044L003.D1'
-Delete '$INSTDIR\levels\C044L003.DAT'
-Delete '$INSTDIR\levels\C044L005.DAT'
-Delete '$INSTDIR\levels\C044L007.D1'
-Delete '$INSTDIR\levels\C044L007.DAT'
-Delete '$INSTDIR\levels\C044L015.D1'
-Delete '$INSTDIR\levels\C044L015.DAT'
-Delete '$INSTDIR\levels\C045L001.DAT'
-Delete '$INSTDIR\levels\C045L002.DAT'
-Delete '$INSTDIR\levels\C045L015.D3'
-Delete '$INSTDIR\levels\C046L001.D1'
-Delete '$INSTDIR\levels\C046L001.D2'
-Delete '$INSTDIR\levels\C046L001.D3'
-Delete '$INSTDIR\levels\C046L001.DAT'
-Delete '$INSTDIR\levels\C046L002.D2'
-Delete '$INSTDIR\levels\C046L002.DAT'
-Delete '$INSTDIR\levels\C046L005.DAT'
-Delete '$INSTDIR\levels\C046L008.DAT'
-Delete '$INSTDIR\levels\C046L009.DAT'
-Delete '$INSTDIR\levels\C046L010.DAT'
-Delete '$INSTDIR\levels\C046L014.DAT'
-Delete '$INSTDIR\levels\C046L015.D1'
-Delete '$INSTDIR\levels\C046L015.D2'
-Delete '$INSTDIR\levels\C046L015.D3'
-Delete '$INSTDIR\levels\C046L015.DAT'
-Delete '$INSTDIR\levels\C047L001.D1'
-Delete '$INSTDIR\levels\C047L001.D2'
-Delete '$INSTDIR\levels\C047L001.D3'
-Delete '$INSTDIR\levels\C047L001.DAT'
-Delete '$INSTDIR\levels\C047L002.DAT'
-Delete '$INSTDIR\levels\C047L011.DAT'
-Delete '$INSTDIR\levels\C047L015.D1'
-Delete '$INSTDIR\levels\C047L015.DAT'
-Delete '$INSTDIR\levels\C049L009.DAT'
-Delete '$INSTDIR\levels\C050L001.D1'
-Delete '$INSTDIR\levels\C050L001.D2'
-Delete '$INSTDIR\levels\C050L001.D3'
-Delete '$INSTDIR\levels\C050L001.DAT'
-Delete '$INSTDIR\levels\C050L002.D1'
-Delete '$INSTDIR\levels\C050L002.D2'
-Delete '$INSTDIR\levels\C050L002.D3'
-Delete '$INSTDIR\levels\C050L002.DAT'
-Delete '$INSTDIR\levels\C050L003.DAT'
-Delete '$INSTDIR\levels\C052L001.DAT'
-Delete '$INSTDIR\levels\C052L007.DAT'
-Delete '$INSTDIR\levels\C060L005.DAT'
-Delete '$INSTDIR\levels\C060L015.DAT'
-Delete '$INSTDIR\levels\C065L001.DAT'
-Delete '$INSTDIR\levels\C065L002.DAT'
-Delete '$INSTDIR\levels\C065L003.DAT'
-Delete '$INSTDIR\levels\C065L005.DAT'
-Delete '$INSTDIR\levels\C070L011.D1'
-Delete '$INSTDIR\levels\C070L011.DAT'
-Delete '$INSTDIR\levels\C079L001.D2'
-Delete '$INSTDIR\levels\C079L002.DAT'
-Delete '$INSTDIR\levels\C079L015.D3'
+Delete $INSTDIR\levels\*
```

Comment:
It may be better to delete C*.DAT, C*.D1, C*.D2, C*.D3.
assuming such wildcards are supported.

Reply:
Yep good call, they are, changed and tested it as well.